extensions/wikia/WAMPage/templates/WAMPageController_index.php

```diff
@@ -1,6 +1,6 @@
 <div class="wam-header wam-hero">
 	<div class="wam-top-gainers-cards">
-		<h2 class="wam-top-gainers-header"><?= wfMessage('wampage-top-gainers-header')->escaped(); ?><? if ( $isSingleVertical ) : ?>: <span class="vertical-<?= Sanitizer::encodeAttribute( $verticalsShorts[ $selectedVerticalId ] ) ?>"><?= $filterVerticals[ $selectedVerticalId ] ?></span><? endif; ?></h2>
+		<h2 class="wam-top-gainers-header"><?= wfMessage('wampage-top-gainers-header')->escaped(); ?><? if ( $isSingleVertical ) : ?>: <span class="vertical-<?= Sanitizer::encodeAttribute( $verticalsShorts[ $selectedVerticalId ] ) ?>"><?= wfMessage( "wam-{$verticalsShorts[$selectedVerticalId]}" )->escaped() ?></span><? endif; ?></h2>
```

Comment:
I'll go with the full names in comments to avoid putting too much logic in the template.